README.md

```diff
@@ -22,15 +22,52 @@ Our goal is to address the following use cases on behalf of customers want to tr
 
 [See It In Action - Sample App](https://github.com/aws-samples/aws-xray-sdk-with-opentelemetry-sample)
 
-## Adding the SDK To Your Project
+### Adding the SDK To Your Project
+
+1. Add snapshot builds and dependencies via your project’s dependency manager
+2. Set the io.opentelemetry.trace.spi.TracerProvider Java property to com.amazonaws.xray.opentelemetry.tracing.TracingProvider
+
+### Getting Started With Maven
+
+```xml
+<project>
+  <repositories>
+    <repository>
+      <id>aws-snapshots</id>
+      <url>https://aws.oss.sonatype.org/content/repositories/snapshots</url>
+    </repository>
+  </repositories>
+  <dependencies>
+    <dependency>
+      <groupId>com.amazonaws</groupId>
+      <artifactId>aws-xray-opentelemetry-sdk-java</artifactId>
```

Comment:
Is this the right artifact? Looking at: https://aws.oss.sonatype.org/content/repositories/snapshots/com/amazonaws/aws-xray-sdk-opentelemetry/